packages/firebase_analytics/firebase_analytics_platform_interface/pubspec.yaml

```
@@ -20,3 +20,4 @@ dev_dependencies:
   firebase_core_platform_interface: ^5.0.0
   flutter_test:
     sdk: flutter
+  pigeon: ^11.0.1
```

```suggestion
  pigeon: 11.0.1
```
We lock the version to prevent having breaking change in other packages since we are generating the pigeons messages in different packages

---

Fixed ✅